python/lsst/dax/imgserv/locateImage.py

```diff
@@ -175,3 +193,15 @@ def __init__(self, config, logger=log):
                        butlerKeys=config["DAX_IMG_BUTLER_KEYS2"],
                        logger=logger)
 
+
+def _get_ds(image_type):
```

Comment:
There's another copy of this function in the cli on :187, although it doesn't return None by default.  Can we get rid of one of these versions?